File: TEST data for large repo/repo-Copy5/Blueprism/BluePrism.Server.Domain.Models/ScheduleLog.vb
Imports Func
Imports System.Runtime.Serialization

<Serializable()>
<DataContract([Namespace]:="bp")>
Public Class ScheduleLog
    <DataMember(Name:="li", EmitDefaultValue:=False)>
    Public Property ScheduleLogId As Integer

    <DataMember(Name:="s", EmitDefaultValue:=False)>
    Public Property Status As ItemStatus

    <DataMember(Name:="t", EmitDefaultValue:=False)>
    Public Property StartTime As [Option](Of DateTime)

    <DataMember(Name:="e", EmitDefaultValue:=False)>
    Public Property EndTime As [Option](Of DateTime)

    <DataMember(Name:="sn", EmitDefaultValue:=False)>
    Public Property ServerName As String

    <DataMember(Name:="i", EmitDefaultValue:=False)>
    Public Property ScheduleId As Integer

    <DataMember(Name:="n", EmitDefaultValue:=False)>
    Public Property ScheduleName As String
End Class

File: TEST data for large repo/repo_Copy-2/Blueprism/BluePrism.Api/BluePrism.Api.Domain/ScheduleLog.cs
namespace BluePrism.Api.Domain
{
    using System;
    using Func;

    public class ScheduleLog
    {
        public int ScheduleLogId { get; set; }
        public Option<DateTime> StartTime { get; set; }
        public Option<DateTime> EndTime { get; set; }
        public ScheduleLogStatus Status { get; set; }
        public string ServerName { get; set; }
        public int ScheduleId { get; set; }
        public string ScheduleName { get; set; }
    }
}
